src/components/layouts/main/index.js

```diff
@@ -0,0 +1,33 @@
+import React, {Component, PropTypes} from 'react'
+import './style.css'
+
+export default class Main extends Component{
+  displayName: 'Main'
+
+  static defaultProps = {
+    isFluid: false
+  }
+
+  render() {
+    const {children, isFluid, backgroundColor} = this.props
+    let className = ''
+    className += (backgroundColor === 'BLUE') ? ' blue' : ''
+    className += isFluid ? ' flexed' : ''
+
+    if (isFluid) {
+      return (
+        <main className={className}>
+          {children}
+        </main>
+      )
+    } else {
+      return (
+        <main className={className}>
+          <div className='container-fluid wrap'>
```

Comment:
This is confusing because 'container-fluid wrap' sounds like a fluid thing to me. Maybe just comment what is going on for now.